### DISCUSSION TITLE
context menu window - Windows 11

### DISCUSSION THREAD

**Discussion by jssta15646461234** [iterate-ch/cyberduck#16185] [Q&A] [answered]
hello, wondering why the context menu (right click on file) didn't bring me the mountainduck options to share a file ? We want to use awss3 as bucket and sharing files outside of the company. thanks,

**Comment by AliveDevil** [comment, marked as answer]
You need to either
- enable the classic context menu permanently,
- shift-right click on a file to get the classic context menu,
- or select More Options in the new context menu
